stripper/ze_forhidden_void_b5.cfg

```diff
@@ -0,0 +1,660 @@
+	}
+	insert:
+	{
+		"OnTrigger" "stage2_zm_end_tpEnable15-1"
+		"OnTrigger" "boss_pushKill15-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"targetname" "author_gift"
+		"classname" "trigger_multiple"
+	}
+	insert:
+	{
+		"filtername" "human"
+	}
+}
+
+
+add:
+{
+	"classname" "trigger_hurt"
+	"damage" "9999999"
+	"damagecap" "999999"
+	"damagemodel" "0"
+	"damagetype" "0"
+	"filtername" "human"
+	"nodmgforce" "0"
+	"origin" "-4062 -5742 -1686"
+	"spawnflags" "1"
+	"StartDisabled" "0"
+	"targetname" "ext"
+	"OnHurtPlayer" "saverunscriptcodecoutex()0-1"
+}
+
+add:
+{
+	"targetname" "addrelay"
+	"classname" "logic_relay"
+	"OnSpawn" "extAddOutputsolid 20.50-1"
+	"OnSpawn" "extAddOutputmins -230 -50 -151.00-1"
+	"OnSpawn" "extAddOutputmaxs 230 50 151.00-1"
+}
+
+add:
+{
+	"classname" "logic_case"
+	"targetname" "sqcase3"
+	"OnCase01" "sqmakerAddOutputorigin -3777.26 -10697.5 -2609.750-1"
+	"OnCase01" "sqmakerForceSpawn0.1-1"
+	"OnCase02" "sqmakerForceSpawn0.1-1"
+	"OnCase02" "sqmakerAddOutputorigin -3677.5 -10697.5 -2609.750-1"
+	"OnCase03" "sqmakerForceSpawn0.1-1"
+	"OnCase03" "sqmakerAddOutputorigin -3586 -10708.5 -2621.750-1"
+	"OnCase04" "sqmakerForceSpawn0.1-1"
+	"OnCase04" "sqmakerAddOutputorigin -3507 -10697.5 -2639.750-1"
+	"OnCase05" "sqmakerForceSpawn0.1-1"
+	"OnCase05" "sqmakerAddOutputorigin -3421 -10702.5 -2653.750-1"
+	"OnCase06" "sqmakerForceSpawn0.1-1"
+	"OnCase06" "sqmakerAddOutputorigin -3783.26 -10575.5 -2609.750-1"
+	"OnCase07" "sqmakerForceSpawn0.1-1"
+	"OnCase07" "sqmakerAddOutputorigin -3683.5 -10575.5 -2609.750-1"
+	"OnCase08" "sqmakerForceSpawn0.1-1"
+	"OnCase08" "sqmakerAddOutputorigin -3592 -10586.5 -2621.750-1"
+	"OnCase09" "sqmakerForceSpawn0.1-1"
+	"OnCase09" "sqmakerAddOutputorigin -3513 -10575.5 -2639.750-1"
+	"OnCase10" "sqmakerForceSpawn0.1-1"
+	"OnCase10" "sqmakerAddOutputorigin -3427 -10580.5 -2653.750-1"
+}
+
+modify:
+{
+	match:
+	{
+		"targetname" "heal_weapon"
+		"classname" "weapon_elite"
+	}
+	delete:
+	{
+		"OnPlayerPickup" "heal_user   AddOutputtargetname Player0-1"
+	}
+	insert:
+	{
+		"OnPlayerPickup" "heal_userAddOutputtargetname Player0-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"targetname" "nrk_item_flamethrower_pistol"
+		"classname" "weapon_knife"
+	}
+	delete:
+	{
+		"OnPlayerPickup" "!activatorAddOutputtargetname naraka_flamethrower0-1"
+	}
+	insert:
+	{
+		"OnPlayerPickup" "!activatorAddOutputtargetname naraka_flamethrower0.05-1"
+		"OnPlayerPickup" "naraka_flamethrowerAddOutputtargetname Player0-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"targetname" "zm_ice_weapon"
+		"classname" "weapon_knife"
+	}
+	delete:
+	{
+		"OnPlayerPickup" "!activatorAddOutputtargetname zombie_ice01"
+	}
+	insert:
+	{
+		"OnPlayerPickup" "!activatorAddOutputtargetname zombie_ice0.05-1"
+		"OnPlayerPickup" "zombie_iceAddOutputtargetname Player0-1"
+	}
+}
```

Comment:
If you're fixing this stuff, you may as well make players use a sane default targetname as well.